Technico/Controllers/AuthController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Mvc;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+using Technico.Services;
+using Technico.Dtos;
+
+[ApiController]
+[Route("[controller]")]
+public class AuthController : ControllerBase
+{
+    private readonly IConfiguration _configuration;
+    private readonly UserService _userService;
+
+    public AuthController(UserService userService, IConfiguration configuration)
+    {
+        _userService = userService;
+        _configuration = configuration;
+    }
+
+    [HttpPost("login")]
+    public async Task<ActionResult<UserResponseDTO>> Login([FromBody] LoginDTO loginModel)
+    {
+        var user = await _userService.ValidateUserByEmail(loginModel.Email, loginModel.Password);
+
+        // Replace this with your actual user validation logic
+        if (user != null)
+        {
+            var token = GenerateJwtToken(user);
+            return Ok(new { User = user, Token = token });
+        }
+
+        return Unauthorized();
+    }
+
+    private string GenerateJwtToken(UserResponseDTO user)
```

Comment:
Consider adding error handling for potential exceptions during user validation and token generation